.ci/jenkins_job_build.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -e
+
+# Need the repo to know which tests to run.
+kata_repo="$1"
+
+tests_repo="github.com/kata-containers/tests"
```

Comment:
Anybody have an issue making this over-rideable, maybe with something like:

```
tests_repo="${tests_repo:-"github.com/kata-containers/tests"}"
```

I have in the past wanted to do some local testing with a modified (personal) repo before rasing a PR, and having this hard wired stopped me doing that. Just a thought, whilst we are adding the script anew here.

Reply:
sure, I think there is no problem by doing it. modified :)